### PR TITLE
fix elixir 1.5 deprecation warnings

### PR DESCRIPTION
**Pull request by mobileoverlord**

### CODE REVIEW

File: lib/nerves_wpa_supplicant/messages.ex [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2014 LKC Technologies, Inc.
+# Copyr©ight 2014 LKC Technologies, Inc.
```
Comment (fhunleth) [inline lib/nerves_wpa_supplicant/messages.ex]: How did you do this? :)

Comment (mobileoverlord) [reply lib/nerves_wpa_supplicant/messages.ex]: no idea

**Comment by fhunleth** [comment]
Not sure if you saw this, but the tests fail. Also, want to update Travis to use 1.5?

**Comment by mobileoverlord** [comment]
yup, taking a look at the test and updating travis.

**Comment by mobileoverlord** [comment]
all good now

**Comment by fhunleth** [comment]
I see that the test failures are gone, but I didn't see the change that fixed it. Will the tests pass under 1.4?

**Comment by mobileoverlord** [comment]
Yeah they will. The change is that trim_trailing/2 requires that the second parameter is a binary. We were using the code point for ? Before. Which is what was causing it to fail.

**Comment by fhunleth** [comment]
Thanks. I see the change now. That was subtle.

**Pull request merged**